app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -21,7 +22,7 @@ body {
 }
 
 .btn-primary-reverse {
-  border: 1px solid $primary !important;
+  border: 2px solid gainsboro !important;
```

Comment:
Isn't there something for this already? 

https://react-bootstrap.github.io/components/buttons/#outline-buttons

Reply:
Ah gotcha